policy/gac_networks.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, n_basis_functions, embed_dim, activation = None):
         Class Args:
```

Comment:
Why this file is changed back to gac_networks.py? Why is it undoing so many code changes?

Reply:
The gac_networks was due to an unupdated branch. An issue with so many people changing the file structure of the app. What other changes did I miss.